docs/overview.md

```diff
@@ -151,6 +151,20 @@ If any change in logic is needed, the Executor/s should be migrated to a new one
 
 <br>
 
+- [PermissionedPayloadsController](../src/contracts/payloads/PermissionedPayloadsController.sol): permissioned version of the payloads controller.
+
+  The execution stage works as follows:
+  - A trusted entity creates any kind of payload and submits it to the permissioned payloads manager.
```

Comment:
Provably not any kind of payload. Will this Payloads controller have a different executor? if so, which permissions will this executor hold? This will determine what kind of payloads will be registered here